_overviews/scala3-book/methods-most.md

```diff
@@ -256,7 +256,7 @@ A `match` expression can also be used as the entire method body, and often is.
 Here’s another version of `isTruthy`, written with a `match` expression :
 
 ```scala
-def isTruthy(a: Any) = a match
+def isTruthy(a: Matchable) = a match
```

Comment:
May I ask to add a comment/link about `Matchable`?

Reply:
Hi @julienrf! I added a short `Matchable` mention and a link to the more complete reference page. Let me know if the phrasing is right.